### PR TITLE
Remove beta guards for JPA default properties feature

### PR DESCRIPTION
**Pull request by dazey3**
for #15979

Signed-off-by: Will Dazey <dazeydev.3@gmail.com>

### CODE REVIEW

File: dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/management/JPAPUnitInfo.java [hunk 1]
```diff
@@ -900,10 +900,8 @@ private EntityManagerFactory createEMFactory(PersistenceUnitInfo puInfo, boolean
         getJPAComponent().addIntegrationProperties(xmlSchemaVersion,
                                                    integrationProperties, ivClassLoader);
 
-        if (ProductInfo.getBetaEdition()) {
-            // Add default persistence properties supplied from JPAComponent configuration
-            getJPAComponent().addDefaultProperties(integrationProperties);
-        }
+        // Add default persistence properties supplied from JPAComponent configuration
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/management/JPAPUnitInfo.java:902]: Please remove the now unused import of com.ibm.ws.kernel.productinfo.ProductInfo;

**Review by tkburroughs — CHANGES_REQUESTED** [review]

**Comment by dazey3** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jpa.tests.jpaconfig_jpa_2.1_fat,com.ibm.ws.jpa.tests.jpaconfig_jpa_2.2_fat,com.ibm.ws.jpa.tests.jpaconfig_jpa_3.0_fat

**Review by tkburroughs — APPROVED** [review]

**Comment by tkburroughs** [comment]
#run_libby

**Comment by tkburroughs** [comment]
#run-libby

**Pull request merged**